### DISCUSSION TITLE
Is there an 'approved' way to clear the database and start over?

### DISCUSSION THREAD

**Discussion by bobkoure** [photoprism/photoprism#1184] [Q&A] [answered]
Long story short, indexing didn't do what I wanted;  I need to reset/clear the database, so I can import the folders I want to make available.
Are there any pointers on how to do that, short of a full uninstall/reinstall?

**Comment by alexisLefebvre** [comment]
For reference, the original issue: https://github.com/photoprism/photoprism-docs/issues/70#issue-853714599

**Comment by kvalev** [comment, marked as answer]
Looking at the [docs](https://github.com/photoprism/photoprism-docs/blob/545f86e113635707de150a502c66e8134c1837a4/docs/getting-started/docker.md#command-reference), you should be able to do that using `photoprism reset`. Alternatively you can always stop all containers and delete the database files.

**Reply by lastzero** [reply]
Right, it's also listed in the command reference of our "Getting Started" docs: https://docs.photoprism.org/getting-started/docker-compose/

**Reply by alexisLefebvre** [reply]
If we reset the database, and we have YAML backups, will we get back all the data (labels, albums, etc.) after reindexing the files?

**Reply by lastzero** [reply]
Not all data, just the photos and albums. Classification in labels will be done via TensorFlow again. The command is going to ask what to delete / reset, incl yaml backups. If you want to keep everything, don't delete your database please :)

**Reply by alexisLefebvre** [reply]
Thanks, I thought it would be a way to reset the stacks without losing the other data, to solve #1178

**Reply by lastzero** [reply]
Stacks essentially are photos / videos. Unstacking means you want a different structure there, which means you need to re-index everything or recursively unstack single files to create additional photos / videos. Even then an issue with (preserving) albums and labels remains: When you unstack a file, will the resulting photo be part of the album (the original photo was in) as well? Should it have the same labels, even though the photo might show a completely different picture?
